eaa/verdictd/README.md

```diff
@@ -1,28 +1,28 @@
-# Verdictd
 
-## Introduction
+# Introduction
 
-Verdictd is a server(as RATS shelterd) which creates an m-TLS(Mutal Transport Layer Security) connection with Attestation Agent via remote attestation (aka "[RA-TLS](https://raw.githubusercontent.com/cloud-security-research/sgx-ra-tls/master/whitepaper.pdf)").
-Mainly functions:
-- Handle "decryption" and "get KEK" requests from Attestation Agent, and response with corresponding results.
-- Launch wrap/unwrap gRPC service which can be used by ocicrypto [containers/ocicrypto](https://github.com/containers/ocicrypt).
-
-## Design
+EAA Verdictd is a remote attestation implementation comprising of a set of building blocks that utilize Intel/AMD Security features to discover, attest, and enable critical foundation security and confidential computing use-cases. 
+It relies on [Rats-tls](https://github.com/alibaba/inclavare-containers/tree/master/rats-tls) to applies the remote attestation fundamentals and standard specifications to maintain a platform data collection service and an efficient verification engine to perform comprehensive trust evaluations. 
+These trust evaluations can be used to govern different trust and security policies applied to any given workload.
 
-TODO
+Verdictd creates an m-TLS(Mutal Transport Layer Security) connection with [Attestation Agent](https://github.com/confidential-containers/attestation-agent) via remote attestation.
+Mainly functions:
+- Implemented EAA protocol which includes "decryption" and "get KEK" requests.
+- Implemented a ocicrypto [containers/ocicrypto](https://github.com/containers/ocicrypt) compatible gRPC service.
+- Implemented a grpc service which can be used to config OPA's policy files.
 
-## Installation
+# Design
 
 TODO
```

Comment:
The [pr](https://github.com/alibaba/inclavare-containers/pull/1495). has provided design doc for verdicd. Please add link .